src/main/java/laddergame/domain/Ladder.java

```diff
@@ -0,0 +1,37 @@
+package laddergame.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Ladder {
+
+    private static final int MIN_HEIGHT = 1;
+
+    private final List<Line> lines;
+
+    public Ladder(final int height, final int numberOfPlayers) {
+        validateNumberOfHeight(height);
+        this.lines = createLines(height, numberOfPlayers);
+    }
+
+    public List<Line> getLines() {
+        return Collections.unmodifiableList(lines);
+    }
```

Comment:
```suggestion
    public List<Line> getLadder() {
        return Collections.unmodifiableList(lines);
    }
```

`toUnmodifiableList` 👍
`toList`가 아닌 `toUnmodifiableList`를 사용하면 어떤 장점이 있을까요?

Reply:
`toUnmodifiableList()` 메서드는 `toList()`메서드와 달리 
반환된 List 객체를 수정불가능하게끔 만들어주고, 
만약 수정을 하게된다면 `UnsupportedOperationException` 예외를 발생시키기 때문에,

Ladder 안의 Line들이 의도치 않게 변경되는 상항을 예방해줄 수 있습니당!